src/zcl_abapgit_branch_overview.clas.abap

```diff
@@ -54,12 +55,18 @@ CLASS zcl_abapgit_branch_overview DEFINITION
     METHODS determine_tags
       RAISING
         zcx_abapgit_exception .
-ENDCLASS.
+    METHODS _sort_commits CHANGING ct_commits TYPE ty_commits.
+    METHODS _get_1st_child_commit
+      IMPORTING itr_commit_sha1s    TYPE tyt_commit_sha1_range
```

Comment:
it_ prefix is what is used in other places, but anyhow, should add it in http://docs.abapgit.org/development/guidelines.html